bukuserver/templates/bukuserver/tags_list.html

```diff
@@ -8,8 +8,10 @@
 
 {% block model_menu_bar_before_filters %}
   {{ super() }}
-  <form id="refresh" class="hidden" method="POST" action="refresh"></form>
-  <li><a href="#" onclick="refresh.submit()">{{ _gettext('Refresh') }}</a></li>
+  <form id="refresh" class="d-none" method="POST" action="refresh"></form>
+  <li class="nav-item">
+    <a class="nav-link" href="#" onclick="refresh.submit()">{{ _gettext('Refresh') }}</a>
+  </li>
```

Comment:
`.hidden` got renamed to match other `display:` classes; also, tab buttons now require `.nav-item`/``.nav-link`